Index: src/css/style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/css/style.css b/src/css/style.css
--- a/src/css/style.css	
+++ b/src/css/style.css	
@@ -32,7 +32,6 @@
     color: greenyellow;
 }
 
-button{
     color: rebeccapurple;
     border-radius: 50px;
 }
@@ -112,8 +111,9 @@
     display: flex;
     list-style-type: none;
     width: 1000px;
-    justify-content: space-around;
-    font-size: 30px;
+    justify-content: space-between;
+    justify-content: left;
+    font-size: 15px;
 }
 
 #flex-container {
Index: src/htmls/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/htmls/index.html b/src/htmls/index.html
--- a/src/htmls/index.html	
+++ b/src/htmls/index.html	
@@ -16,6 +16,7 @@
 
             <li><a href="languages.html">Frontend Basics</a></li>
             <li><a href="scripts.html"  >Scripts</a></li>
+            <li><a href="Crypto.html"  >Crypto Ranks</a></li>
             <li><a href="AboutMe.html"  >About me</a></li>
 
         </ul>
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	
+++ b/.idea/misc.xml	
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="openjdk-19" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="15" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
   <component name="ProjectType">
Index: src/css/crypto.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/css/crypto.css b/src/css/crypto.css
new file mode 100644
--- /dev/null	
+++ b/src/css/crypto.css	
@@ -0,0 +1,3 @@
+body {
+    margin: 30px;
+}
\ No newline at end of file
Index: src/js/crypto.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/js/crypto.js b/src/js/crypto.js
new file mode 100644
--- /dev/null	
+++ b/src/js/crypto.js	
@@ -0,0 +1,43 @@
+var baseUrl = "https://api.coinranking.com/v2/coins";
+var proxyUrl = "https://cors-anywhere.herokuapp.com/";
+var apiKey = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx";
+
+var apiUrl = `${proxyUrl}${baseUrl}`;
+console.log(apiUrl);
+
+fetch(`${proxyUrl}${baseUrl}`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+      'X-My-Custom-Header': `${apiKey}`,
+      'Access-Control-Allow-Origin': "*"
+    }
+})
+  .then((response) => {
+    if (response.ok) {
+      response.json().then((json) => {
+        console.log(json.data);
+        let coinsData = json.data.coins;
+
+        if (coinsData.length > 0) {
+          var cryptoCoin = "";
+        }
+        //For Loop Starts
+        coinsData.forEach((coin) => {
+          cryptoCoin += "<tr>";
+          cryptoCoin += `<td> ${coin.uuid} </td>`;
+          cryptoCoin += `<td> ${coin.btcPrice} </td>`;
+          cryptoCoin += `<td> ${coin.rank}</td>`;
+          cryptoCoin += `<td> ${coin.tier} </td>`;
+          cryptoCoin += `<td> ${coin.name}</td>`;
+          cryptoCoin += `<td> $${Math.round(coin.price)} Billion</td>`;
+          cryptoCoin += `<td> ${coin.symbol}</td>`;"<tr>";
+        });
+        //For Loop Ends
+        document.getElementById("data").innerHTML = cryptoCoin;
+      });
+    }
+  })
+  .catch((error) => {
+    console.log(error);
+  });
\ No newline at end of file
Index: src/htmls/Crypto.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/htmls/Crypto.html b/src/htmls/Crypto.html
new file mode 100644
--- /dev/null	
+++ b/src/htmls/Crypto.html	
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+  <link rel="stylesheet" href="crypto.css" />
+
+  <title>Document</title>
+</head>
+<body>
+
+<nav class="navbar fixed-top navbar-light bg-dark">
+  <div class="container-fluid">
+    <a class="navbar-brand text-light">Crypto Dashboard</a>
+    <form class="d-flex">
+      <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success" type="submit">Search</button>
+    </form>
+  </div>
+</nav>
+
+
+<table class="table table-bordered" style="margin-top: 70px" >
+  <thead>
+  <tr>
+    <th scope="col">UUID</th>
+    <th scope="col">Btc Price</th>
+    <th scope="col">Rank</th>
+    <th scope="col">Tier</th>
+    <th scope="col">Name</th>
+    <th scope="col">Price</th>
+    <th scope="col">Symbol</th>
+
+  </tr>
+  </thead>
+  <tbody id="data">
+  </tbody>
+</table>
+<script src="crypto.js"></script>
+</body>
+</html>
\ No newline at end of file
